doc/Makefile

```
@@ -7,29 +7,25 @@ SPHINXAUTOGEN = sphinx-autogen
 BUILDDIR      = _build
 
 # Internal variables.
-ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees  $(SPHINXOPTS) .
+ALLSPHINXOPTS = -d $(BUILDDIR)/doctrees  $(SPHINXOPTS) .
 
-.PHONY: help clean html linkcheck doctest api
-
-all: html
+.PHONY: help api html server clean
 
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
```

```suggestion
	@echo "Please use 'make <target>' where <target> is one of"
```
I think it looks better to have the two quote types match